### PR TITLE
refactor(attach): Used meta direct as key, simplified a bit

### PR DESCRIPTION
**Pull request by rhuss**

### CODE REVIEW

File: pkg/syndesis/operation/attach.go [hunk 1]
```diff
@@ -52,47 +52,27 @@ func getAllManagerSelectors() []metav1.ListOptions {
 		{
 			LabelSelector: "app=syndesis,syndesis.io/app=todo",
 		},
-		// In the 7.0 template, the database does not have syndesis.io/type=infrastructure label
-		// It can be removed when it's no longer necessary to upgrade from 7.0 (together with the "legacy" package)
-		{
-			LabelSelector: "syndesis.io/app=syndesis,syndesis.io/component=syndesis-db",
```
Comment (nicolaferraro) [inline pkg/syndesis/operation/attach.go:58]: Is the label fixed in current 7.0 installations? Otherwise we should leave this.

**Review by nicolaferraro — COMMENTED** [review]
Didn't notice this one. Seems much simpler :+1: 

Just 1 comment..

File: pkg/syndesis/operation/attach.go (hunk 1 above)
Comment (rhuss) [reply pkg/syndesis/operation/attach.go:58]: Yes, it's fixed and merged --> https://github.com/syndesisio/syndesis/pull/3163

going to move this PR over to Syndesis and lets merge it there.

**Comment by rhuss** [comment]
Obosoleted by https://github.com/syndesisio/syndesis/pull/3193

**Pull request closed without merging**